ubclaunchpad/ubc-degree-navigator
# Refactored textract calls to support transfer credit upload

## Describe your changes
This PR adds endpoint to parse transfer credit file upload, returning in JSON format. The endpoint is /api/user/transfer
#### Screenshot (if applicable)
<img width="1018" alt="image" src="https://github.com/ubclaunchpad/ubc-degree-navigator/assets/16951211/72baf464-a13f-49f6-910f-4e933c0e61de">

Screenshot of postman request included above. 

## How do I test
> Provide brief steps on how to test these changes. 
1. Pull this branch into local and run go mod tidy. 
2. Add .env file in src folder, and paste credentials in format AWS_ACCESS_KEY_ID  = abcdef12345 (without quotation marks). Message me for credentials if I've not sent you these. 
3. In Postman, create a request as shown in the screenshot. Ensure that the key, value (file) and description are all non-empty, and that all headers are checked in the header tab. The expected result should match the format in the screenshot 

## Checklist before requesting a review
